externals/simplecpp/simplecpp.cpp

```diff
@@ -757,6 +757,7 @@ void simplecpp::TokenList::constFoldUnaryNotPosNeg(simplecpp::Token *tok)
                 tok->setstr(tok->op + tok->next->str);
                 deleteToken(tok->next);
                 break;
+            default: ;
```

Comment:
1. remove all changes in simplecpp and tinyxml.
if you want to update simplecpp then the repo is http://github.com/danmar/simplecpp
tinyxml is also on GitHub somewhere
most of your changes are invalid as the goal is to only use C++03. I want it to be as portable as possible in case somebody wants to reuse it.

2. I am not sure why this default is needed. please either explain that or remove it.